src/components/Contactus.jsx

```diff
@@ -0,0 +1,8 @@
+const Contact = () => {
+  return (
+    <>
+    </>
+  );
+};
+
+export default Contact;
```

Comment:
- [ ] Please fill this section as mentioned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website_v2.md). Your app should have atleast three pages.